app/importers/api_importer.rb

```diff
@@ -0,0 +1,16 @@
+require 'rubygems'
+require 'HTTParty'
```

Comment:
@nihonjinrxs 
I'm pretty sure this should be `require 'httparty'`

it's working on your mac (I assume) because mac's hard drive's are case insentive, but in prod on linux its case sensitive.

Reply:
Ah, gotcha.  Will mod and see if that fixes it.